src/murmur/main.cpp

```diff
@@ -648,11 +648,7 @@ int main(int argc, char **argv) {
 
 	meta->getOSInfo();
 
-	Version::component_t major, minor, patch;
-	QString strver;
-	meta->getVersion(major, minor, patch, strver);
-
-	qWarning("Murmur %d.%d.%d (%s) running on %s: %s: Booting servers", major, minor, patch, qPrintable(strver),
+	qWarning("Murmur %s running on %s: %s: Booting servers", qPrintable(Version::toString(Version::get())),
```

Comment:
Why not ``Version::getRelease()``?

Reply:
Hm, yeah I guess that'd make more sense :+1: